src/waveform/visualplayposition.cpp

```diff
@@ -104,25 +108,29 @@ double VisualPlayPosition::determinePlayPosInLoopBoundries(
         if (loopSize > 0) {
             if ((data.m_playRate < 0.0) &&
                     (interpolatedPlayPos < data.m_loopStartPos) &&
-                    (data.m_playPos >= data.m_loopStartPos)) {
+                    (data.m_playPos >= data.m_loopStartPos) &&
+                    !data.m_loopInAdjustActive) {
                 // 1. Deck playing reverse
                 // 2. Interpolated playposition at the time of next VSync would
                 // be outsite of the active loop
                 // 3. Playposition is currently inside the active loop
                 //    (not scratching left of an activated loop)
+                // 4. LoopIn is not hold down
```

Comment:
minor English nit (and below)
```suggestion
                // 4. LoopIn is not being held down
```